mongo-oplog/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// TypeScript Version: 2.1
+// Type definitions for mongo-oplog v2.0.2
+// Project: https://github.com/cayasso/mongo-oplog
+// Definitions by: FinalDes <https://github.com/FinalDes>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="eventemitter3" />
+
+export function MongoOplog(url: string, filter: any): OplogTrigger;
```

Comment:
According to the documentation, the second parameter is [`options`](https://github.com/cayasso/mongo-oplog#mongooploguri-options).